### DISCUSSION TITLE
Change disgestion site or trypsin

### DISCUSSION THREAD

**Discussion by BenoitBe** [vdemichev/DiaNN#1186] [Q&A]
Dear Vadim,

Is it possible to ask trypsin(enzyme) to cleave only after R (and not K) in order to process a biochemical modified sample (library generation & process)?

Thx in advance,

Benoit

**Comment by vdemichev** [comment]
Hi Benoit,

Yes, please just type "--cut R* " (without quotes) in Additional options.

Best,
Vadim

**Comment by BenoitBe** [comment]
So just slection of Trypsin/P as proteasse and "--cut R* ". Sound good!

If I can (and I think you already had this kind of questions), if I want to add a specific modif not in unimod, I can just add the corresponding amino acid and modification of mass in additional options?

Thx again :)

**Reply by vdemichev** [reply]
Yes, modification names can be arbitrary, do not need to match unimod
